packages/server-core/src/projects/scene/scene.class.ts

```diff
@@ -190,18 +191,18 @@ export class SceneService
       )
     )
     try {
-      await storageProvider.createInvalidation(sceneAssetFiles.map((asset) => `${directory}${newSceneName}${asset}`))
+      await storageProvider.createInvalidation(SCENE_ASSET_FILES.map((asset) => `${directory}${newSceneName}${asset}`))
     } catch (e) {
       logger.error(e)
-      logger.info(sceneAssetFiles)
+      logger.info(SCENE_ASSET_FILES)
     }
 
     if (isDev) {
       const projectPathLocal = path.resolve(appRootPath.path, localDirectory)
-      for (const ext of sceneAssetFiles) {
+      for (const ext of SCENE_ASSET_FILES) {
         fs.copyFileSync(
-          path.resolve(appRootPath.path, `${localDirectory}default${ext}`),
-          path.resolve(projectPathLocal + newSceneName + ext)
+          path.resolve(appRootPath.path, `${DEFAULT_DIRECTORY}/default${ext}`),
```

Comment:
@barankyle, I think `DEFAULT_DIRECTORY` should be used in `getDirectoryFromData` method of `scene.hooks.ts` at line 66. It should be used to set `data.localDirectory`.